api/lib/mno_enterprise/concerns/mailers/system_notification_mailer.rb

```diff
@@ -211,6 +211,21 @@ def access_approved(user_access_request_id)
     )
   end
 
+  def access_approved_all(user_id, requester_id, access_duration)
```

Comment:
Is it possible to pass the objects `user` and `requester` as they have already been loaded before?

Reply:
I don't trust serialization on BaseResource Object.